ts/Core/Chart/Chart.ts

```diff
@@ -3521,6 +3521,52 @@ class Chart {
         fireEvent(this, 'selection', { resetSelection: true }, this.zoom);
     }
 
+    /**
+     * Checks if mouse pointer is within pane.
+     *
+     * @private
+     * @function Highcharts.Chart#isWithinPane
+     * @emits Highcharts.Chart#event:afterIsWithinPane
+     */
+    public isWithinPane(
```

Comment:
We already have `chart.isInsidePlot`. See if we can use that directly instead of adding a function that is quite similar.

Reply:
Perhaps the method should be called `isInsideAxis`? `isInsidePlot` checks if `x`+`y` position is within the plotting area. Meanwhile `isInsideAxis` checks if `x` or `y` is within `xAxis` or `yAxis`. This is used for example in multiple panes, like Stock, and zooming on yAxis (when selecting area within one pane, then only that one yAxis/pane should zoom in). Demo: https://www.highcharts.com/demo/stock/stock-tools-gui (select yAxis zoom from StockTools and zoom in on both panes)

Reply:
Can you check that again? `isInsidePlot` does lok at `series.xAxis` and `series.yAxis`, and uses axis.pos and axis.len to determine wheter the point is within. That is the same as looking at axis.left, top, width and height. I suspect that `isInsidePlot` can be used either directly or with small modifications.

Reply:
The `isInsidePlot` function also has `ignoreX` and `ignoreY` options so you can use it to inspect one axis at the time.